cmd/kyaml/cmd/merge.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/kyaml/kio"
+	"sigs.k8s.io/kustomize/kyaml/kio/filters"
+)
+
+func GetMergeRunner() *MergeRunner {
+	r := &MergeRunner{}
+	c := &cobra.Command{
+		Use:   "merge [SOURCE_DIR...] [DESTINATION_DIR]",
+		Short: "Merge Resource configuration files",
+		Long: `Merge Resource configuration files
+
+Merge reads Kubernetes Resource yaml configuration files from stdin or sources packages and write
+the result to stdout or a destination package.
+
+Resources are merged using the Resource [apiVersion, kind, name, namespace] as the key.  If any of
+these are missing, merge will default the missing values to empty.
+
+Resources specified later are high-precedence (the source) and Resources specified
+earlier are lower-precedence (the destination).
+
+Merge uses the following rules for merging a source Resource into a destination Resource:
```

Comment:
Does this line need to be moved or is this a TODO?

Reply:
Should be removed and followed up on:

https://github.com/kubernetes-sigs/kustomize/issues/1756

Reply:
Added